src/pretix/base/models/items.py

```diff
@@ -799,6 +801,13 @@ class ItemVariation(models.Model):
         help_text=_('If set, this will be displayed next to the current price to show that the current price is a '
                     'discounted one. This is just a cosmetic setting and will not actually impact pricing.')
     )
+    require_approval = models.BooleanField(
+        verbose_name=_('Buying this product requires approval'),
```

Comment:
I’d say, we should make it consistent. Directly above it is a checkbox „This variation will only be shown if a voucher matching …“ Switching to „product“ then reads a bit weird. Another alternative might be to shorten it to something like „Require approval“ (similar to the „Require a valid membership“ further down the page). The help-text could even be shortened to something like: „Buying this will put the order into an "approval" state …“

I think the help-text above „This variation will only be shown if a voucher matching the product is redeemed.“ has an error and should read „This variation will only be shown if a voucher matching the products’ variation is redeemed.“ Or maybe shorter: „only show if a matching voucher is redeemed“?

Shortening all these checkboxes makes the language more consistent. Currently all three: „This variation will only be shown if a voucher matching the product is redeemed.“, „Buying this product requires approval“ and „Require a valid membership“ have a different type of _language_.